augur/filter.py

```diff
@@ -80,6 +80,22 @@ def run(args):
             to_exclude = set([line.strip() for line in ifile if line[0]!=comment_char])
         seq_keep = [s for s in seq_keep if s not in to_exclude]
 
+    if args.exclude_where:
+        to_exclude = []
+        for ex in args.exclude_where:
+            col, val = ex.split("=")
+            print(col, val)
+            for seq_name in seq_keep:
+                group = []
+                if seq_name not in meta_dict:
+                    print("WARNING: no metadata for %s, skipping"%seq_name)
+                    continue
+                else:
+                    m = meta_dict[seq_name]
+                    if m[col] == val:
```

Comment:
Probably good to guard this dictionary key access with `m.get(col)` so that it doesn't raise a KeyError if `col` is missing.